README.md

```diff
@@ -81,7 +86,7 @@ jsonApi.define({
 
 jsonApi.start();
 ```
-Your new API will be alive at `http://localhost:16006/` and your `photos` resources will be at `http://localhost:16006/photos`.
+Your new API will be alive at `http://localhost:16006/` and your `photos` resources will be at `http://localhost:16006/photos`. The GraphiQL interface will be available at `http://localhost:16006/graphql`.
```

Comment:
It's a thing, if you check the URL you'll be met by an *i*nteractive webapp :+1: